src/main/kotlin/io/github/rybalkinsd/kohttp/ext/RequestExt.kt

```diff
@@ -0,0 +1,45 @@
+package io.github.rybalkinsd.kohttp.ext
+
+import okhttp3.Headers
+import okhttp3.Request
+import okhttp3.RequestBody
+import okio.Buffer
+import java.nio.charset.Charset
+
+/**
+ *  Build curl command of the request
+ *
+ * @author doyaaaaaken
+ */
+internal fun Request.buildCurlCommand(): String {
+    return buildString {
+        append("curl -X ${method()}")
+        append(buildCurlHeaderOption(headers()))
+        append(buildCurlBodyOption(body()))
+        append(" \"${url()}\"")
+    }
+}
+
+private fun buildCurlHeaderOption(headers: Headers): String {
+    return headers.asSequence().map { header ->
+        val value = header.value.let { v ->
+            if (v.startsWith('"') && v.endsWith('"')) {
+                v.substring(1, v.length - 1)
+            } else {
+                v
+            }
+        }
+        " -H \"${header.name}: $value\""
+    }.joinToString("")
+}
+
+private fun buildCurlBodyOption(body: RequestBody?): String {
+    return if (body == null) {
+        ""
+    } else {
+        val buffer = Buffer().apply { body.writeTo(this) }
+        val utf8 = Charset.forName("UTF-8")
+        val charset = body.contentType()?.charset(utf8) ?: utf8
+        " --data $'" + buffer.readString(charset).replace("\n", "\\n") + "'"
```

Comment:
Could you please share why we need to replace `\n` here?

Also looks like `utf-8` should be introduced as a const for both HTTP and CURL at least

Reply:
> utf-8 should be introduced as a const for both HTTP and CURL at least

In case of HTTP, `ByteString.utf8()` method is used currently.
https://github.com/rybalkinsd/kohttp/blob/93336018d342c3d6858d3af4445b5df01de9f7ac/src/main/kotlin/io/github/rybalkinsd/kohttp/interceptors/LoggingInterceptor.kt#L49

And in case of CURL, we can use `Buffer.readUtf8()` method in stead of assigning `Charset` class.
Which do you think better?

1. Use `Charset` class. So, I fix logic of HTTP case.
2. Use `Buffer.readUtf8()` method. So, I fix logic of CURL case.

Reply:
> Could you please share why we need to replace \n here?

Because we can output curl command in one-liner.

For example, if we wouldn't replace `\n`, we would output below curl command.
```bash
curl -X POST --data $'{
>   "aa": "bb",
>   "aa1": "bb2"
> }' "https://postman-echo.com/post"
```

But , with replacing `\n`, we could output command as one-liner.
```bash
curl -X POST --data $'{\n  "aa": "bb",\n  "aa1": "bb2"\n}' "https://postman-echo.com/post"
```

Reply:
I see, make sence